runconfig/hostconfig.go

```diff
@@ -96,38 +109,46 @@ func (n PidMode) Valid() bool {
 	return true
 }
 
+// DeviceMapping represents the device mapping between the host and the container
 type DeviceMapping struct {
 	PathOnHost        string
 	PathInContainer   string
 	CgroupPermissions string
 }
 
+// RestartPolicy represents the restart policies of the container
 type RestartPolicy struct {
 	Name              string
 	MaximumRetryCount int
 }
 
+// IsNone indicates whether the container has no restart policy
```

Comment:
Oh wait, perhaps what you meant was "no" as the type of policy - if so, just quote the word "no".